### ISSUE TITLE
Unable to select time icon and date icon using keyboard arrow keys-  For Accessibility 

### ISSUE THREAD

**Issue by midhun8454** [Eonasdan/bootstrap-datetimepicker#2328]
This repo is no longer actively monitor or supported.

Please read this [blog post](https://eonasdan.com/state-of-my-picker)
